src/client/audiomanager.js

```diff
@@ -13,11 +16,29 @@ AudioManager.playBGM = async function(name) {
       return;
     }
     const [filename, child] = name.split('/');
+    console.log(filename, child);
     const wzNode = await WZManager.get(`Sound.wz/${filename}.img/${child}`);
     this.bgm = wzNode.nGetAudio();
     this.bgm.loop = true;
     this.bgm.play();
+
+    //Debug
+    this.setMusicVolume(0.01);
+    this.setSFXVolume(0.01);
   }
 };
 
+AudioManager.setMusicVolume = async function (vol) {
```

Comment:
The `async` is not necessary because you're not using `await` in the body of the function. Also, don't do type checking here; it's an antipattern and the caller is expected to pass a number.

Reply:
Got it, I was trying to make some sort of validation